planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -51,11 +51,12 @@ robot_model_loader::RobotModelLoader::RobotModelLoader(const Options &opt)
   configure(opt);
 }
 
-robot_model_loader::RobotModelLoader::~RobotModelLoader()
-{
+robot_model_loader::RobotModelLoader::~RobotModelLoader() {
   model_.reset();
   rdf_loader_.reset();
   kinematics_loader_.reset();
+
+  ROS_DEBUG( "Robot model loader destructor" );
```

Comment:
I don't really know if they are needed, I used them to find the bug so I thought they might be of use to someone in the future.